### ISSUE TITLE
Recreate Horton's clover field

### ISSUE THREAD

**Issue by kostmo** [swarm-game/swarm#1440]
[Horton](https://en.wikipedia.org/wiki/Horton_Hears_a_Who!) needs to find one clover among millions.
Could use the new world-generation DSL for the map.  The target clover should be randomly placed.

The scenario would entail maximizing search efficiency.

![horton-hears-a-who-2008-720p-brrip-x264-yify20-15-30](https://github.com/swarm-game/swarm/assets/261693/603c5dbb-36a7-42c3-b8bc-5a711c459b50)

**Comment by byorgey** [comment]
Interesting, I'm not sure with the current DSL there's a way to randomly choose a single location.  I'll have to think about what primitive feature(s) we could add to support that.

**Comment by kostmo** [comment]
> not sure with the current DSL there's a way to randomly choose a single location

Well, probably a system robot can handle this.

**Comment by byorgey** [comment]
> Well, probably a system robot can handle this.

Yes, it could, I was just taking it as inspiration for making the DSL more expressive.
